package.json

```diff
@@ -75,7 +75,7 @@
     "exists-sync": "0.0.3",
     "fake-xml-http-request": "^1.4.0",
     "faker": "3.0.1",
-    "pretender": "^1.2.0",
+    "pretender": "^1.3.0",
```

Comment:
right but `master` is now on 1.4 so if you remove this & rebase we should be set